dynamic_analysis_pkg/setup.py

```diff
@@ -0,0 +1,14 @@
+from setuptools import setup
+
+setup(name='dynamic_analysis_pkg',
+      version='0.1.7',
+      description='Machinary to fetch simulation results to a Jupyter Notebook',
+      author='Stefano Antonel',
+      author_email='stefano.antonel@epfl.ch',
```

Comment:
Up to you but I usually put:
```suggestion
    author="BlueBrain NSE",
    author_email="bbp-ou-nse@groupes.epfl.ch",
```
But it might be different for the mooc. I don't know.

Reply:
Should we apply here our Python guidelines?